live_flights_on_map.md

```diff
@@ -0,0 +1,125 @@
+## Documentation for live flight details on a map
+
+##### 1. Map Creation
+
+Firstly, we have to insert OpenLayers css into an HTML file:
+
+```html
+<link rel="stylesheet"href="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/css/ol.css"type="text/css">
+```
+
+Then we have to insert OpenLayers js file:
+
+```html
+<script src="https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.1.1/build/ol.js"></script>
+```
+
+Inside `body` of the html file, we have to create a `map` \<div>:
```

Comment:
@arnabdutta73 can you provide a link to the html docs for `<div>` tag?  This will help the users without web development experience to understand the use of the tag.

Reply:
No need for detailed explanation, just a hyperlink to the documentation for the tag.

Reply:
write up assumes basic understanding of HTML.